specification/kubernetesconfiguration/resource-manager/readme.go.md

```diff
@@ -8,6 +8,13 @@ go:
   clear-output-folder: true
 ```
 
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: package-2019-11-01-preview
```

Comment:
why do we need multi-api for only one tag?

Reply:
When generating go SDK, the automation pipeline will run autorest with the option `multiapi`, if there is no multiapi section in the config, nothing will be generated